examples/shopping_ads/add_performance_max_retail_campaign.py

```diff
@@ -263,11 +267,14 @@ def _create_performance_max_campaign_operation(
     # Set the Final URL expansion opt out. This flag is specific to
     # Performance Max campaigns. If opted out (True), only the final URLs in
     # the asset group or URLs specified in the advertiser's Google Merchant
-    # Center or business data feeds are targeted.
+    # Center or business data feeds are targeted. This is the recommended
+    # way for accounts that use their Merchant Center feed. As this is an
+    # example of PMax for Retail that uses a Merchant Center feed, we will
+    # set True the URL expansion opt out.
```

Comment:
Just so that this example is consistent with the .NET one, can you remove this additional text, and add the below right above the line where `url_expansion_opt_out` is set?

```
# For a Retail campaign, we want the final URL's to be limited to those explicitly surfaced via GMC.
```

Reply:
will do!